src/components/TooltipButton.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactElement } from "react";
+import { svgSrc } from "@/helpers";
+
+import SVG from "react-inlinesvg";
+
+import { Avatar, IconButton, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import { HtmlTooltip } from "@/components/HtmlTooltip";
+import { theme } from "@/theme";
+
+interface Props {
+  tooltip: string;
+  svgSrc: string;
+  size?: "small" | "large" | "inline";
+  onClick?: (event?: React.MouseEvent<HTMLButtonElement>) => void;
+  isActive?: boolean;
+  type?: "button" | "submit";
+}
+
+const useStyles = makeStyles({
+  iconButton: {
+    padding: ({ size }: Partial<Props>) => {
+      if (size === "small") return "4px";
+      if (size === "large") return "5px 11px";
+
+      return "2px";
+    },
+  },
+  svg: {
+    width: "22px",
+    height: "100%",
+    marginLeft: "-1px",
+    fill: ({ isActive }: Partial<Props>) =>
```

Comment:
Can you embed @SilviaZeta's fix for button hovering? Your still show green on green when selected?

Reply:
i was about to ask her about it, is it in annotate?

Reply:
I believe so